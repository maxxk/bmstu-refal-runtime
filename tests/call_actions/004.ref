$ENTRY Go
{
	= <Prout <Func 'Begin Middle End'>>
}

Func
{
	e.1 e.2, e.1 -> {'Begin' = Success} = e.1;
}



