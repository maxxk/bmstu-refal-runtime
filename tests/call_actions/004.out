[VM]: Func call failed!
Func call term:
<AnonymFunc_2 >
Field of view:
<Prout <Func Begin Middle End[<AnonymFunc_2 >]>>
